workflows/learning_curve_regression.py

```diff
@@ -125,7 +125,7 @@ def run(self):
 
         # remove validation data from dataset for finite datasets
         if self._data.is_finite:
-            remaining_data = self._data.complement(validation_data)
+            remaining_data = self._data._complement(self._data,validation_data)
```

Comment:
```suggestion
            remaining_data = self._data.complement(self._data,validation_data)
```
Thanks for the contribution! You're right that the complement method requires two arguments. I do think that we want to use `complement` instead of `_complement`, since the former does exception handling.